scripts/check-users-in-space-developer-role.ps1

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env pwsh
+
+param([String] $space, [String] $slackwebhook)
+
+$data = @()
+
+$bearer = $(cf oauth-token)
```

Comment:
If we have access to `cf` isn't it a lot easier to just do:

```
cf space-users dfe $space | awk '/SPACE DEVELOPER/, /SPACE AUDITOR/' | sed 1d | head -n -2
```

That's sh syntax, but you get the idea ... no need for HTTP rest requests I don't think.

Reply:
(sorry, I tried to submit this yesterday but flubbed it up. Point still stands, wouldn't this all be easier just using `cf`?)

Reply:
it started off like that or least similar to that...... but the apiV3 offers more flexibility and propensity to evolve better.